### PR TITLE
added font directory path and size to site health

### PR DESCRIPTION
**Pull request by andr3ribeiro**
Added the font directory in the array for site health table iteration

Trac ticket: https://core.trac.wordpress.org/ticket/60719

### CODE REVIEW

**Comment by ironprogrammer** [comment]
Thanks for the PR, @andr3ribeiro! Perhaps a good addition to this would be to include the fonts dir "is writable" check found on the Filesystem Permissions tab. This work is something we could probably get merged early this cycle.

File: src/wp-admin/includes/class-wp-debug-data.php [hunk 1]
```diff
@@ -482,6 +482,15 @@ public static function debug_data() {
 					'value' => $loading,
 					'debug' => 'loading...',
 				),
+				'fonts_path'   => array(
+					'label' => __( 'Fonts directory location' ),
+					'value' => wp_get_font_dir()['path'],
```
Comment (peterwilsoncc) [inline src/wp-admin/includes/class-wp-debug-data.php]:
```suggestion
					'value' => wp_get_font_dir()['basedir'],
```

File: src/wp-admin/includes/class-wp-debug-data.php [hunk 2]
```diff
@@ -1636,6 +1645,7 @@ public static function get_sizes() {
 			'themes_size'    => get_theme_root(),
 			'plugins_size'   => WP_PLUGIN_DIR,
 			'uploads_size'   => $upload_dir['basedir'],
+			'fonts_size'     => wp_get_font_dir()['path'],
```
Comment (peterwilsoncc) [inline src/wp-admin/includes/class-wp-debug-data.php]:
```suggestion
			'fonts_size'     => wp_get_font_dir()['basedir'],
```

**Review by peterwilsoncc — CHANGES_REQUESTED** [review]
Similar to the uploads directory tests, this should use the `basedir` value rather than `path`.

While they are the same on a default WordPress install, plugins may filter the path for individual uploads by adding a sub-directory, for example:

```php
array (
  'path' => '/path/to/wp-content/uploads/fonts/font-family',
  'url' => 'http://example.com/wp-content/uploads/fonts/font-family',
  'subdir' => '/font-family',
  'basedir' => '/path/to/wp-content/uploads/fonts',
  'baseurl' => 'http://example.com/wp-content/uploads/fonts',
  'error' => false,
)
```

**Comment by audrasjb** [comment]
I committed the changes suggested by @peterwilsoncc directly to the PR.

**Review by audrasjb — APPROVED** [review]

**Comment by audrasjb** [comment]
Committed in https://core.trac.wordpress.org/changeset/58299

**Pull request closed without merging**